src/renderer/components/MessageWrapper.js

```diff
@@ -241,19 +242,23 @@ function convertMessageStatus (s) {
 }
 
 function convertContentType (message) {
-  var filemime = message.filemime
-  if (!filemime) return 'image/jpg'
-  if (filemime === 'application/octet-stream') {
-    switch (message.msg.viewType) {
-      case C.DC_MSG_IMAGE:
-        return 'image/jpg'
-      case C.DC_MSG_VOICE:
-        return 'audio/ogg'
-      default:
-        return 'application/octect-stream'
-    }
+  const filemime = message.filemime
+
+  if (!filemime) return 'application/octet-stream'
+  if (filemime !== 'application/octet-stream') return filemime
+
+  switch (message.msg.viewType) {
+    case C.DC_MSG_IMAGE:
+      return 'image/jpg'
+    case C.DC_MSG_VOICE:
+      return 'audio/ogg'
```

Comment:
We can probably simplify this switch statement to a simple `mime.lookup()` which should catch `'image/jpg'` and `'audio/ogg'` as well but it's fine for now I guess.